submissions/krylenger/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,1342 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body class="body">
+    <main class="container-half-screen container-half-screen--film-info">
+      <header class="film-info__header">
+        <h1 class="film-info__title">eX machina</h1>
+        <h2 class="film-info__subtitle">Drama, Sci-Fi, Thriller</h2>
+        <h2 class="film-info__rate">9.4</h2>
+      </header>
+      <section class="film-info__main">
+        <h6 class="film-info__description">
+          A young programmer is selected to participate in a ground-breaking experiment in synthetic intelligence by
+          evaluating the human qualities of a highly advanced humanoid A.I.
+        </h6>
```

Comment:
Avoid abrupt heading level changes. In this case better to use `p`.
Same problem with BEM blocks at line 16. [Read more](https://ru.bem.info/methodology/quick-start/#%D0%BF%D1%80%D0%B8%D0%BD%D1%86%D0%B8%D0%BF-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%8B-%D1%81-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B0%D0%BC%D0%B8)